x-pack/agent/pkg/fleetapi/client_test.go

```diff
@@ -15,10 +15,42 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/elastic/beats/agent/kibana"
+	"github.com/elastic/beats/agent/release"
 	"github.com/elastic/beats/x-pack/agent/pkg/config"
+	"github.com/elastic/beats/x-pack/agent/pkg/core/logger"
 )
 
 func TestHTTPClient(t *testing.T) {
+	t.Run("Ensure we validate the remote Kibana version is higher or equal",
```

Comment:
this tests just that we are passing kbn-version right? not actual check

Reply:
Yes exactly